wrappers/winuser.go

```diff
@@ -379,3 +382,47 @@ func SetThreadDesktop(desktop syscall.Handle) error {
 	}
 	return nil
 }
+
+func RegisterWindowMessage(messageName *uint16) (uint32, error) {
+	r1, _, e1 := syscall.Syscall(procRegisterWindowMessageW.Addr(), 1, uintptr(unsafe.Pointer(messageName)), 0, 0)
+	if r1 == 0 {
+		if e1 != ERROR_SUCCESS {
+			return 0, e1
+		} else {
+			return 0, syscall.EINVAL
+		}
+	}
+	return uint32(r1), nil
+}
+
+func SendNotifyMessage(hwnd syscall.Handle, msg uint32, wparam, lparam uintptr) error {
+	r1, _, e1 := syscall.Syscall6(
+		procSendNotifyMessageW.Addr(),
+		4,
+		uintptr(hwnd),
+		uintptr(msg),
+		wparam,
+		lparam,
+		0,
+		0)
+	if r1 == 0 {
+		if e1 != ERROR_SUCCESS {
+			return e1
+		} else {
+			return syscall.EINVAL
+		}
+	}
+	return nil
+}
+
+func BlockInput(blockIt bool) error {
+	r1, _, e1 := syscall.Syscall(procBlockInput.Addr(), 1, boolToUintptr(blockIt), 0, 0)
+	if r1 == 0 {
+		if e1 != ERROR_SUCCESS {
+			return e1
+		} else {
+			return syscall.EINVAL
+		}
+	}
+	return nil
+}
```

Comment:
Let's put these new functions in alphabetical order with the rest.

Reply:
Corrected.